static/src/js/containers/DatepickerContainer/DatepickerContainer.jsx

```diff
@@ -139,7 +173,6 @@ class DatepickerContainer extends Component {
     // A valid date will come be passed as an ISO string. Check to see if the date is a valid ISO string,
     // if so, we convert it to a UTC string in our desired format. If the value is not a valid ISO date,
     // then we leave it untouched and pass it to the input.
-    // We are using YYYY-MM-DDTHH:m:s.SSSZ instead of moment.ISO_8601 so that it doesn't autocomplete every time there's a valid ISO format
```

Comment:
This comment is actually valid now since you changed it back to  'YYYY-MM-DDTHH:mm:ss.SSSZ' we should keep it since its informative for the future